apps/web/src/locales/en/messages.json

```
@@ -6227,5 +6227,31 @@
   },
   "errorReadingImportFile": {
     "message": "An error occurred when trying to read the import file"
+  },
+  "createSecret": {
+    "message": "Create a secret"
+  },
+  "createProject": {
+    "message": "Create a project"
+  },
+  "createServiceAccount": {
+    "message": "Create a service account"
+  },
+  "downloadThe": {
+    "message": "Download the",
+    "description": "Link to a downloadable resource. This will be used as part of a larger phrase. Example: Download the Secrets Manager CLI"
+  },
+  "smCLI": {
+    "message": "Secrets Manager CLI"
+  },
+  "importSecrets": {
+    "message": "Import secrets"
+  },
+  "getStarted": {
+    "message": "Get started"
+  },
+  "complete": {
+    "message": "Complete",
+    "description": "Example: 1/4 Complete"
```

Thoughts on doing. Should make it more descriptive what it's used for. Assumes we aren't doing any styling in the component though.

```suggestion
  "complete": {
    "message": "$COMPLETED$/$TOTAL$ Complete",
    "placeholders": {
      "COMPLETED": {
        "content": "$1",
        "example": "1"
      },
      "TOTAL": {
        "content": "$2",
        "example": "4"
      }
    }
```